chat/public/js/src/chat.js

```diff
@@ -394,16 +383,6 @@ function ChatXBlock(runtime, element, init_data) {
      * user data passed from the backend
      */
     var init = function() {
```

Comment:
+1 to removing the spacer, it wasn't helping anywhere I could see.  But this comment below made me wonder if it was really necessary?

Reply:
The "prevent rubber band effect (overscroll) in iOS app" comment only applies to the `position: fixed; overflow: hidden;` css tweak below, it does not apply to the spacer. It looks like I forgot to move the comment to correct place after I [added the code related to the spacer](https://github.com/open-craft/xblock-chat/commit/c1f4e9cdb6563f612731ca660eb8635e4a9552dc#diff-1da48437e00ac09f195adf29177ba523R378).

The position: fixed; tweaks that was used to fix the overscroll efect in previous version of the iOS app actually does the opposite in the latest app. It causes weird scrolling issues. Removing it fixes the problem. It still allows overscroll, but so do other types of modules/blocks, so I don't think that's a problem.